src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,102 @@
+            key={todo.id}
+            data-cy="todo"
+            className={classNames(
+              {
+                'has-background-info-light':
+                currentTodo?.id === todo.id,
+              },
+            )}
+          >
+            <td className="is-vcentered">{todo.id}</td>
             <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">quo adipisci enim quam ut ab</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+              {todo.completed && (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
                 </span>
-              </button>
+              )}
             </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
             <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
+              {/* eslint-disable-next-line max-len */}
+              <p className={todo.completed ? 'has-text-success' : 'has-text-danger'}>
+                {todo.title}
+              </p>
             </td>
-
             <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
+              <button
+                data-cy="selectButton"
+                className="button"
+                type="button"
+                onClick={() => dispatch(currentTodoActions.setTodo(todo))}
+              >
                 <span className="icon">
-                  <i className="far fa-eye" />
+                  {/* eslint-disable-next-line max-len */}
+                  <i className={currentTodo?.id === todo.id ? 'far fa-eye-slash' : 'far fa-eye'} />
```

Comment:
use classNames for this
```suggestion
                  <i className={cn('far fa-eye', {
                      'fa-eye-slash': currentTodo?.id === todo.id
                    })}
                 />
```

Reply:
It doesn't work with classNames this way. Maybe I'm doing something wrong... 
implemented it with classNames with two conditions but I think ternary operator variant was better